rln/src/protocol.rs

```diff
@@ -544,13 +541,13 @@ pub fn generate_proof_with_witness(
     proving_key: &(ProvingKey<Curve>, ConstraintMatrices<Fr>),
 ) -> Result<ArkProof<Curve>, ProofError> {
     // If in debug mode, we measure and later print time take to compute witness
-    #[cfg(debug_assertions)]
+    #[cfg(test)]
```

Comment:
Not sure that is good idea to replace this on test. They have different behaviour. Why are you change it?

Reply:
This is in accordance with "5. Modify the code related to testing and measurements to facilitate studying the performance of the witness calculation for the **release compilation mode**." Measurements in the debug mode in general may yield more pessimistic results compared to the real ones.